mkdocs.yml

```diff
@@ -60,7 +60,7 @@ nav:
       - Configurations: 'config.md'
       - Limitations of v1: 'limits.md'
 
-  - Development - Use Case:
+  # - Development - Use Case:
```

Comment:
Probably `Use Cases` could be the subsection under `Development - Intermeditate`